pkg/phlaredb/compact.go

```diff
@@ -215,10 +217,16 @@ func newBlockWriter(dst string, meta *block.Meta, rewriterFn SymbolsRewriterFn)
 		return nil, err
 	}
 
+	downsampler, err := downsample.NewDownsampler(blockPath)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I think we should only initialize it if `meta.Compaction.Level > 2`, otherwise we create empty tables for downsampled data